### ISSUE TITLE
L'utilisateur peut chercher un arrêt de bus par une recherche de texte.

### ISSUE THREAD

**Issue by maxbodin** [maxbodin/bus-stop-explorer#5]

**Comment by Bastien2203** [comment]
Pourquoi pas par ligne de bus aussi ça peux être pratique.
Et ca serais bien peut etre de pouvoir masquer certaines lignes sur la carte, ou d'avoir des filtres parce que quand il y a tout les arrets c'est un peu chargé

**Comment by maxbodin** [comment]
Yes, j'approuve la recherche pour les lignes & je te laisse faire un autre issue pour la possibilité d'utiliser des filtres ?
